### DISCUSSION TITLE
Command PhaseScriptExecution failed with a nonzero exit code XCode

### DISCUSSION THREAD

**Discussion by mohammednabil95** [firebase/flutterfire#12928] [Q&A]
I upgraded firebase versions and all current firebase packages version is 15.0.0 and i also upgraded to 13.0 of iOS deployment target based on changelog. But when i try to "Archive" from XCode then it is showing an error "command phasescriptexception failed with a nonzero exit code". Can someone help me to fix this issue during archive.
Note: It is working fine if i run the app from Xcode or Android Studio 

```
Unhandled exception:
Exception: 
#0      UploadCrashlyticsSymbols.run (package:flutterfire_cli/src/commands/upload_symbols.dart:406:7)
<asynchronous suspension>
#1      CommandRunner.runCommand (package:args/command_runner.dart:212:13)
<asynchronous suspension>
#2      main (file:///Users/md-nabel/.pub-cache/hosted/pub.dev/flutterfire_cli-1.0.0/bin/flutterfire.dart:63:5)
<asynchronous suspension>
Command PhaseScriptExecution failed with a nonzero exit codeUnhandled exception:
```

**Comment by zkrige** [comment]
tick "for install builds only"

<img width="390" alt="image" src="https://github.com/firebase/flutterfire/assets/968412/173078d1-2b80-4480-a9f3-fba129370d00">

**Reply by Cyrille-Dakhlia** [reply]
Hi @zkrige ,
By ticking it, my Flutter app now runs on my iOS simulator (even though I still don't receive any logs into Crashlytics and I can't figure out why yet)

Could you explain what does it do exactly?

**Reply by Cyrille-Dakhlia** [reply]
I believe it simply did not run the script which feels more like a workaround than a real solution, isn't it?

**Comment by Cyrille-Dakhlia** [comment]
@mohammednabil95 you can check [this thread](https://github.com/invertase/flutterfire_cli/issues/293)
